src/app/home/components/DataList/Clusters/ClusterContent.tsx

```diff
@@ -33,7 +36,14 @@ const ClusterContent = ({
       ) : (
         <Flex alignItems="center" justifyContent="center">
           <Box>
-            <DataListEmptyState type="cluster" />
```

Comment:
did away with the DataListEmptyState component b/c it wasnt really reducing any boilerplate